pennylane_lightning/lightning_tensor/quimb/_mps.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
```

Comment:
I think I'd prefer if this lived inside a `backends` directory, though I'll defer that for now if there's good reason to the contrary.